server/models/db/worker.js

```diff
@@ -11,6 +11,21 @@ async function getWorker(email, password) {
     return userModel.getUser(email, password, Tables.worker, Worker);
 }
 
+// This method inserts a worker account given specific information.
+async function insertWorkerAccount(first_name, last_name, type, worker_id, email, password, phone, specialization) {
```

Comment:
Hi Nancy, 
Same comment as mentioned above to adjust the method parameter naming to lowerCamelCase.

Reply:
Hi Amy,
Thank you for clarifying that!
My action: I've changed the method parameter names to firstName, lastName, and workerId.
You can now see these changes in line 15 of server/models/db/worker.js

Please confirm if this takes care of your suggestion!

Reply:
Hi Nancy, 
Yes, that works. Thank you for updating those parameters!